apps/src/templates/EligibilityChecklist.jsx

```diff
@@ -1,222 +0,0 @@
-/** @file Maker Discount Code Eligibility Checklist */
-import React, {Component, PropTypes} from 'react';
-import i18n from "@cdo/locale";
-import Button from "./Button";
-import SchoolAutocompleteDropdownWithLabel from './census2017/SchoolAutocompleteDropdownWithLabel';
-
-import ValidationStep, {Status} from '../lib/ui/ValidationStep';
-
-export default class EligibilityChecklist extends Component {
-  static propTypes = {
-    statusPD: PropTypes.oneOf(Object.values(Status)).isRequired,
-    statusStudentCount: PropTypes.oneOf(Object.values(Status)).isRequired,
-    unit6Intention: PropTypes.string,
-  };
-
-  state = {
-    statusYear: Status.UNKNOWN,
-    yearChoice: null, // stores the teaching-year choice until submitted
-    submitting: false,
-    displayDiscountAmount: false,
-    submission: {
-      name: '',
-      email: '',
-      role: '',
-      country: 'United States',
-      hoc: '',
-      nces: '',
-      schoolName: '',
-      schoolCity: '',
-      schoolState: '',
-      schoolZip: '',
-      schoolType: '',
-      afterSchool: '',
-      tenHours: '',
-      twentyHours: '',
-      otherCS: false,
-      followUpFrequency: '',
-      followUpMore: '',
-      acceptedPledge: false
```

Comment:
most of these fields, we (a) dont care about and (b) wont be changing anyways. in the new code here https://github.com/code-dot-org/code-dot-org/pull/19357/files#diff-76833f7fc3ce9199e3654082fb20c69aR34 we only look for nces.